packages/amplify_core/lib/src/types/exception/api/invalid_url.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_core/src/types/exception/api/api_exception.dart';
+
+/// {@template amplify_core.api.api_invalid_url_exception}
+/// Exception thrown when an invalid URL is provided to the request
+/// {@endtemplate}
+class ApiInvalidURLApiException extends ApiException {
```

Comment:
I only see this being used once - is this needed? Or should it be used in more places?

Reply:
Removed in favor of `ConfigurationError`